Config/services.php

```diff
@@ -5,8 +5,6 @@
 use MauticPlugin\SparkpostBundle\Mailer\Factory\SparkpostTransportFactory;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
-use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
```

Comment:
We need to use this function in the imports. If we remove this I get this error of Attempted to call function "service" from the global namespace.